extensions/Cite/Cite.php

```diff
@@ -87,10 +87,10 @@ function wfCite( $parser ) {
 
 /**
  * @param $out OutputPage
- * @param $sk Skin
+ * @param Skin $skin
  * @return bool
  */
-function wfCiteBeforePageDisplay( $out, &$sk ) {
+function wfCiteBeforePageDisplay( OutputPage $out, Skin $skin ): bool {
```

Comment:
@slayful — a crippled program does more damage than a dead one. It is a good practice to crash early.  If we pay for `Skin` byt get `null`, we should crash and investigate rather than desperately try to handle the situation. :-)  A massive PHP fatal error will be automatically reported and JIRA case will be created. In my opinion this is a good change.